quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/Main.java

```diff
@@ -33,7 +33,10 @@
                 "",
                 "Find more information at: https://www.keycloak.org/docs/latest"
         },
-        description = "%nUse this command-line tool to manage your Keycloak cluster.",
+        description = {
+                "Use this command-line tool to manage your Keycloak cluster.",
```

Comment:
@andreaTP Could you please send another PR including a `%n` in the begining of the description. I missed that during my review. It broke the format.